librb/include/rb_include.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _RB_INCLUDE_H
+#define _RB_INCLUDE_H 1
+
+// This is exactly how they recommend doing it in the docs by Alan Smithee,
+// but the gcc docs on the other hand, with tongue in cheek, tell the tale
+// by metaphor: https://gcc.gnu.org/onlinedocs/cpp/Computed-Includes.html
+
+#include <RB_INC_ASSERT_H
```

Comment:
It fails, but the point is you can #ifdef that somewhere else like in configure.ac so if the header isn't available it just #include's /dev/null